diff --git a/dev/alpine/main/python39/APKBUILD b/dev/alpine/main/python39/APKBUILD
index 82a0505..f4f4f20 100644
--- a/dev/alpine/main/python39/APKBUILD
+++ b/dev/alpine/main/python39/APKBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 # Contributor: Sheila Aman <sheila@vulpine.house>
 
-pkgname=python3
+pkgname=python39
 # the python3-tkinter's pkgver needs to be synchronized with this.
 pkgver=3.9.18
 _bluez_ver=5.55
diff --git a/dev/alpine/main/python310/APKBUILD b/dev/alpine/main/python310/APKBUILD
index 2540d2d..3f093cf 100644
--- a/dev/alpine/main/python310/APKBUILD
+++ b/dev/alpine/main/python310/APKBUILD
@@ -3,7 +3,7 @@
 # Original Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 # Original Contributor: Sheila Aman <sheila@vulpine.house>
 
-pkgname=python3
+pkgname=python310
 # the python3-tkinter's pkgver needs to be synchronized with this.
 pkgver=3.10.14
 _bluez_ver=5.65
diff --git a/dev/alpine/main/python311/APKBUILD b/dev/alpine/main/python311/APKBUILD
index 4da17cf..529998d 100644
--- a/dev/alpine/main/python311/APKBUILD
+++ b/dev/alpine/main/python311/APKBUILD
@@ -3,7 +3,7 @@
 # Original Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 # Original Contributor: Sheila Aman <sheila@vulpine.house>
 
-pkgname=python3
+pkgname=python311
 # the python3-tkinter's pkgver needs to be synchronized with this.
 pkgver=3.11.9
 _basever="${pkgver%.*}"
diff --git a/dev/alpine/main/python312/APKBUILD b/dev/alpine/main/python312/APKBUILD
index c5110d2..7d89af4 100644
--- a/dev/alpine/main/python312/APKBUILD
+++ b/dev/alpine/main/python312/APKBUILD
@@ -3,7 +3,7 @@
 # Original Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 # Original Contributor: Sheila Aman <sheila@vulpine.house>
 
-pkgname=python3
+pkgname=python312
 # the python3-tkinter's pkgver needs to be synchronized with this.
 pkgver=3.12.5
 _basever="${pkgver%.*}"
