diff --git a/dev/alpine/main/python39/APKBUILD b/dev/alpine/main/python39/APKBUILD
index f4f4f20..6898a9e 100644
--- a/dev/alpine/main/python39/APKBUILD
+++ b/dev/alpine/main/python39/APKBUILD
@@ -87,7 +87,6 @@ build() {
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
 		--enable-optimizations \
-		--enable-shared \
 		--with-lto \
 		--with-computed-gotos \
 		--with-dbmliborder=gdbm:ndbm \
diff --git a/dev/alpine/main/python310/APKBUILD b/dev/alpine/main/python310/APKBUILD
index 3f093cf..ae1a53b 100644
--- a/dev/alpine/main/python310/APKBUILD
+++ b/dev/alpine/main/python310/APKBUILD
@@ -119,7 +119,6 @@ build() {
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
 		--enable-optimizations \
-		--enable-shared \
 		--with-lto \
 		--with-computed-gotos \
 		--with-dbmliborder=gdbm:ndbm \
diff --git a/dev/alpine/main/python311/APKBUILD b/dev/alpine/main/python311/APKBUILD
index 529998d..a64db14 100644
--- a/dev/alpine/main/python311/APKBUILD
+++ b/dev/alpine/main/python311/APKBUILD
@@ -126,7 +126,6 @@ build() {
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
 		--enable-optimizations \
-		--enable-shared \
 		--with-lto \
 		--with-computed-gotos \
 		--with-dbmliborder=gdbm:ndbm \
diff --git a/dev/alpine/main/python312/APKBUILD b/dev/alpine/main/python312/APKBUILD
index 7d89af4..59722d0 100644
--- a/dev/alpine/main/python312/APKBUILD
+++ b/dev/alpine/main/python312/APKBUILD
@@ -124,7 +124,6 @@ build() {
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
 		--enable-optimizations \
-		--enable-shared \
 		--with-lto \
 		--with-computed-gotos \
 		--with-dbmliborder=gdbm:ndbm \
