diff --git a/dev/alpine/main/python39/APKBUILD b/dev/alpine/main/python39/APKBUILD
index 010de68..42e20e0 100644
--- a/dev/alpine/main/python39/APKBUILD
+++ b/dev/alpine/main/python39/APKBUILD
@@ -14,9 +14,26 @@ url="https://www.python.org/"
 arch="all"
 license="PSF-2.0"
 subpackages="$pkgname-dbg $pkgname-dev $pkgname-doc $pkgname-tests::noarch"
-makedepends="expat-dev openssl1.1-compat-dev zlib-dev ncurses-dev bzip2-dev xz-dev
-	sqlite-dev libffi-dev tcl-dev linux-headers gdbm-dev>=1.18.1-r0 readline-dev
-	mpdecimal-dev automake autoconf libtool autoconf-archive !gettext-dev"
+makedepends="
+	!gettext-dev
+	autoconf
+	autoconf-archive
+	automake
+	bzip2-dev
+	expat-dev
+	gdbm-dev>=1.18.1-r0
+	libffi-dev
+	libtool
+	linux-headers
+	mpdecimal-dev
+	ncurses-dev
+	openssl-dev
+	readline-dev
+	sqlite-dev
+	tcl-dev
+	xz-dev
+	zlib-dev
+	"
 source="https://www.python.org/ftp/python/$pkgver/Python-$pkgver.tar.xz
 	https://www.kernel.org/pub/linux/bluetooth/bluez-$_bluez_ver.tar.xz
 	fix-xattrs-glibc.patch
