diff --git a/dev/alpine/main/python310/APKBUILD b/dev/alpine/main/python310/APKBUILD
index 83ad956..09f79c8 100644
--- a/dev/alpine/main/python310/APKBUILD
+++ b/dev/alpine/main/python310/APKBUILD
@@ -51,6 +51,12 @@ source="https://www.python.org/ftp/python/$pkgver/Python-$pkgver.tar.xz
 	CVE-2024-8088.patch
 	"
 options="net" # Required for tests
+prepare_py_provides() {
+	:
+} # disables python inference
+scan_python3_dependency() {
+	:
+} # disables python inference
 builddir="$srcdir/Python-$pkgver"
 
 # secfixes:
@@ -201,7 +207,7 @@ EOF
 }
 
 package() {
-	make -j1 DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
+	make -j `nproc` DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
 	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 	# those are provided by python3-tkinter
 	rm -r "$pkgdir"/usr/bin/idle* "$pkgdir"/usr/lib/python*/idlelib \
diff --git a/dev/alpine/main/python311/APKBUILD b/dev/alpine/main/python311/APKBUILD
index 78715e9..bb1f56e 100644
--- a/dev/alpine/main/python311/APKBUILD
+++ b/dev/alpine/main/python311/APKBUILD
@@ -50,6 +50,12 @@ source="https://www.python.org/ftp/python/$pkgver/Python-$pkgver.tar.xz
 	CVE-2024-8088.patch
 	"
 options="net" # Required for tests
+prepare_py_provides() {
+	:
+} # disables python inference
+scan_python3_dependency() {
+	:
+} # disables python inference
 builddir="$srcdir/Python-$pkgver"
 
 # secfixes:
@@ -195,7 +201,7 @@ EOF
 }
 
 package() {
-	make -j1 DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
+	make -j `nproc` DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
 	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 
 	install -Dm644 "$srcdir"/externally-managed \
diff --git a/dev/alpine/main/python312/APKBUILD b/dev/alpine/main/python312/APKBUILD
index 2ff2f1d..6ef64e7 100644
--- a/dev/alpine/main/python312/APKBUILD
+++ b/dev/alpine/main/python312/APKBUILD
@@ -50,6 +50,12 @@ source="https://www.python.org/ftp/python/$pkgver/Python-$pkgver.tar.xz
 	CVE-2024-8088.patch
 	"
 options="net" # Required for tests
+prepare_py_provides() {
+	:
+} # disables python inference
+scan_python3_dependency() {
+	:
+} # disables python inference
 builddir="$srcdir/Python-$pkgver"
 
 # secfixes:
@@ -177,7 +183,7 @@ EOF
 }
 
 package() {
-	make -j1 DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
+	make -j `nproc` DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
 	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 
 	install -Dm644 "$srcdir"/externally-managed \
diff --git a/dev/alpine/main/python39/APKBUILD b/dev/alpine/main/python39/APKBUILD
index 23c7976..b5bc86e 100644
--- a/dev/alpine/main/python39/APKBUILD
+++ b/dev/alpine/main/python39/APKBUILD
@@ -45,6 +45,12 @@ source="https://www.python.org/ftp/python/$pkgver/Python-$pkgver.tar.xz
 	gnu-fallback-soabi.patch
 	"
 options="net" # Required for tests
+prepare_py_provides() {
+	:
+} # disables python inference
+scan_python3_dependency() {
+	:
+} # disables python inference
 builddir="$srcdir/Python-$pkgver"
 
 # secfixes:
@@ -184,7 +190,7 @@ EOF
 }
 
 package() {
-	make -j1 DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
+	make -j `nproc` DESTDIR="$pkgdir" EXTRA_CFLAGS="$CFLAGS" install maninstall
 	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
 	# those are provided by python3-tkinter
 	rm -r "$pkgdir"/usr/bin/idle* "$pkgdir"/usr/lib/python*/idlelib \
